central/views/imagecve/view_impl.go

```diff
@@ -96,29 +96,37 @@ func (v *imageCVECoreViewImpl) Get(ctx context.Context, q *v1.Query, options vie
 	}
 
 	var err error
-	q, err = common.WithSACFilter(ctx, resources.Image, q)
+	// Avoid changing the passed query
+	cloned := q.Clone()
+	cloned, err = common.WithSACFilter(ctx, resources.Image, cloned)
 	if err != nil {
 		return nil, err
 	}
 
 	var cveIDsToFilter []string
-	if q.GetPagination().GetLimit() > 0 || q.GetPagination().GetOffset() > 0 {
+	if cloned.GetPagination().GetLimit() > 0 || cloned.GetPagination().GetOffset() > 0 {
 		// TODO(@charmik) : Update the SQL query generator to not include 'ORDER BY' and 'GROUP BY' fields in the select clause (before where).
```

Comment:
Suggest we write a ticket under https://issues.redhat.com/browse/ROX-24279 so we don't lose track of it

Reply:
Done